src/lib/viewers/image/ImageViewer.js

```diff
@@ -44,6 +48,10 @@ class ImageViewer extends ImageBaseViewer {
         this.imageEl.classList.add(CLASS_INVISIBLE);
 
         this.currentRotationAngle = 0;
+
+        if (this.options.enableAnnotationsImageDiscoverability) {
```

Comment:
Seems like this should be cleaned up in `ImageViewer.destroy` instead of in `ImageBaseViewer`